### DISCUSSION TITLE
iOS通过pod方式导入'libpag'，报错Undefined symbol: _OBJC_CLASS_$_PAGImageView，请问如何解决？

### DISCUSSION THREAD

**Discussion by li-qingnan** [Tencent/libpag#1566] [General]
iOS通过pod方式导入'libpag'，报错Undefined symbol: _OBJC_CLASS_$_PAGImageView，请问如何解决？

**Comment by FusionXu** [comment]
是首次使用pag，还是升级pag引入的问题？
可以看看我们的demo，https://github.com/libpag/pag-ios
还是有问题的话，可以制作一个可以复现这个问题的demo，传到discussion上。discussion是支持附件直接拖进来的

**Reply by li-qingnan** [reply]
我这边是首次使用，新建了一个demo，pod只导入libpag，是没有问题的。
我觉得是和别的pod库有冲突吧，但是不确定是哪一个？ 请问能给分析一下和什么pod冲突吗？

**Reply by li-qingnan** [reply]
补充说明，模拟器运行正常，真机运行就会报错。
Undefined symbol: _OBJC_CLASS_$_PAGImageView

**Reply by Hparty** [reply]
回复下使用的版本库，我们在demo中check下PAGImageView存不存在

**Reply by kevingpqi123** [reply]
7 日内没有回复更多的信息，信息不全，先关闭该问题
